custom_components/elasticsearch/es_doc_creator.py

```diff
@@ -330,8 +332,13 @@ def state_to_document(self, state: State, time: datetime, version: int = 2) -> d
         """
 
         document_body = {
-            "@timestamp": time_tz,
+            "@timestamp": time_tz.isoformat(),
             "hass.object_id": state.object_id,
+            "event": {
+                "action": reason,
+                "type": "change",
```

Comment:
If "Publish all" is enabled, then we will include docs that don't represent state/attribute changes. Does `"type": "change"` make sense for those?

Reply:
Yeah `info` make sense to me